Lib/platform.py

```diff
@@ -329,20 +333,31 @@ def _linux_distribution(distname, version, id, supported_dists,
 
     """
     try:
-        etc = os.listdir(_UNIXCONFDIR)
-    except OSError:
-        # Probably not a Unix system
+        _os_release = open(_UNIXCONFDIR)
```

Comment:
ResourceWarning will be emitted here.

Why you use underscored names for local variables?